src/Psy/CodeCleaner/ValidClassNamePass.php

```diff
@@ -171,8 +175,12 @@ protected function ensureClassExists($name, $stmt)
     /**
      * Ensure that a referenced interface exists.
      *
-     * @param string $name
-     * @param Stmt   $stmt
+     * @param $interfaces
+     * @param Stmt $stmt
+     *
+     * @throws FatalErrorException
+     *
+     * @internal param string $name
```

Comment:
oh sorry, my IDE must had auto completed that one, just skip it. I didn't meant to add this one.